app/notion/errors.py

```diff
@@ -0,0 +1,2 @@
+class DashboardException(Exception):
```

Comment:
I think Merelo wanted exceptions next to the classes we create, but oh well, we don't have classes yet LOL so these are "system exceptions".